FAQ.md

```diff
@@ -0,0 +1,66 @@
+# FAQ
+
+Answers to frequently asked questions about the [cleanvision](https://github.com/cleanlab/cleanvision) open-source package.
+
+The code snippets in this FAQ come from a fully executable notebook you can run via Colab or locally by downloading it [here](https://github.com/cleanlab/cleanlab/blob/master/docs/source/tutorials/faq.ipynb).
+
+
+1. **What kind of machine learning tasks can I use cleanvision for?**
+
+CleanVision is independent of any machine learning tasks as it directly works on images and does not require and labels or metadata to detect issues in the dataset. The issues detected by CleanVision are helpful for all kinds of machine learning tasks.
+
+2. **Can I check for specific issues in my dataset?**
+
+Yes, you can specify issues like `light` or `blurry` in the issue_types argument when calling `imagelab.find_issues`
+
+```python
+imagelab.find_issues(issue_types={"light": {}, "blurry": {}})
+```
+
+3. What dataset formats does CleanVision support?
+
+Apart from plain image files, CleanVision also works with HuggingFace and Torchvision datasets. You can use the dataset objects as is with the `image_key` argument.
+
+```python
+imagelab = Imagelab(hf_dataset=dataset, image_key="image")
+```
+
+
+
+For more detailed usage instructions and examples, please refer to the [official documentation](https://cleanvision.readthedocs.io/).
+
+## Known issues
+
+1. **Encountering error from `multiprocessing` library while running cleanvision.**
+
+```shell
+    This probably means that you are not using fork to start your
+    child processes and you have forgotten to use the proper idiom
+    in the main module:
+
+        if __name__ == '__main__':
+            freeze_support()
+            ...
+
+    The "freeze_support()" line can be omitted if the program
+    is not going to be frozen to produce an executable.
+
+    To fix this issue, refer to the "Safe importing of main module"
+    section in https://docs.python.org/3/library/multiprocessing.html
+```
+The above issue is caused by multiprocessing module working differently for macOS and Windows platforms. A detailed discussion of the issue can be found [here](https://github.com/cleanlab/cleanlab/issues/159).
+A fix around this issue is to run cleanvision in the main namespace like this
+```python
+if __name__ == "__main__":
+
+    imagelab = Imagelab(data_path)
+    imagelab.find_issues()
+    imagelab.report()
+```
+OR use `n_jobs=1`
```

Comment:
```suggestion
OR use `n_jobs=1` to disable parallel processing:
```

Reply:
always try to explain what a potential solution is changing in order to work around the problem. In this case, there is clear downside of this solution, so it is important to explain it is disabling parallel processing